Packs/Cybereason/Integrations/Cybereason/Cybereason.py

```diff
@@ -1910,6 +1912,66 @@ def get_sensor_id_command(client: Client, args: dict):
         return CommandResults(readable_output=f"Available Sensor IDs are {output}")
 
 
+def fetch_machine_details_command(client: Client, args: dict):
+    machine_name = str(args.get('machineName'))
+    json_body = fetch_machine_details_command_pagination_params(args)
```

Comment:
You didn't add the machine_name to the request.

Reply:
We are not adding it to the API request, we are filtering the data after we have received the response